go/src/meguca/parser/commands.go

```diff
@@ -56,7 +56,6 @@ func parseCommand(match []byte, board string, id uint64, ip string) (com common.
 		com.Type = common.Pyu
 
 		if boardConfig.Pyu {
-			now := time.Now().UTC()
 			tx, err := db.StartTransaction()
```

Comment:
This function is prone to shadowning issues of the error variable with db.RollbackOnError(). Could you use db.InTransaction instead? I'll remove those functions from the master branch in a sec.